scripts/ipfs/install

```diff
@@ -29,9 +29,25 @@ ipfs init || true
 # Enable gossipsub routing
 ipfs config Pubsub.Router gossipsub
 
+# Enable relay hop to help peers with connecting
+ipfs config --bool Swarm.EnableRelayHop true
```

Comment:
This would increase bandwidth usage on internet connected peers.

Also how does this work when you block internet peers.  Will that mean that a non-internet enabled peer will not want to relay through an internet relayed one?

Reply:
> This would increase bandwidth usage on internet connected peers.

Possibly, but only if we manually set up the relay to be used. The current release does not automatically use relays. With this, we have the option too, and it will improve the overall health of the network.

> Also how does this work when you block internet peers. Will that mean that a non-internet enabled peer will not want to relay through an internet relayed one?

As was said in chat by me:
> Whether it will allow the connection to a blacklisted peer over a relay is a good question though, something to be tested. Just keep in mind IPFS won't try to do that on its own just yet

Relaying through an Internet peer would actually be a good thing for nodes to do, but we will have to wait until IPFS does it automatically, possibly in the next release. Once that happens, we can likely remove all the filtering entirely.